sql/src/main/java/io/crate/execution/engine/collect/sources/InformationSchemaIterables.java

```diff
@@ -125,6 +130,50 @@ public InformationSchemaIterables(final Schemas schemas,
         // these are initialized on a clusterState change
         routines = emptyList();
         clusterService.addListener(this);
+
+        pgIndices = () -> tablesStream(schemas).filter(this::isPrimaryKey).map(this::pgIndex).iterator();
+
+        pgClasses = () -> concat(sequentialStream(relations).map(this::relationToPgClassEntry),
+                                 sequentialStream(primaryKeys).map(this::primaryKeyToPgClassEntry)).iterator();
+
+    }
+
+    private boolean isPrimaryKey(RelationInfo relationInfo) {
+        return relationInfo != null && (relationInfo.primaryKey().size() > 1 ||
+                                        (relationInfo.primaryKey().size() == 1 &&
+                                         !relationInfo.primaryKey().get(0).name().equals("_id")));
```

Comment:
```suggestion
        return relationInfo.primaryKey().size() > 1 ||
            (relationInfo.primaryKey().size() == 1 &&
             !relationInfo.primaryKey().get(0).name().equals("_id"));
```

I think we don't have `null` `RelationInfo` entries.